src/qrules/conservation_rules.py

```diff
@@ -692,8 +692,8 @@ def spin_conservation(
 
 
 def spin_magnitude_conservation(
-    ingoing_spins: List[SpinEdgeInput],
-    outgoing_spins: List[SpinEdgeInput],
+    ingoing_spin_magnitudes: List[EdgeQN.spin_magnitude],
+    outgoing_spin_magnitudes: List[EdgeQN.spin_magnitude],
     interaction_qns: SpinMagnitudeNodeInput,
```

Comment:
Something I realize now is that this rule seems to be checking two things and therefore needs $L$ and $S$ of the nodes. It would be better to split that.

Maybe we should write an issue for this if this rule can be split up.

Reply:
This is also not really a suitable class name:
https://github.com/ComPWA/qrules/blob/a045be5fbe29454a92b24739b6bc41dfa42c1c13/src/qrules/conservation_rules.py#L467-L470

Reply:
Yeah seems fishy to me. @grayson-helmholz could you post an issue to split this rule into spin magnitude and $LS$ magnitude.